test/tools/libtesteth/TestSuite.h

```diff
@@ -47,6 +49,12 @@ class TestSuite
 	// Copy .json tests from the src folder to the dest folder because such test is crafted manually and could not be filled.
 	// Used in bcForgedTest, ttWrongRLP tests and such.
 	void copyAllTestsFromFolder(std::string const& _testFolder) const;
+
+	// Return full path to folder for tests from _testFolder
+	fs::path getFullPathFiller(std::string const& _testFolder) const;
+
+	// Structure  <suiteFolder>/<testFolder>/<test>.json
+	fs::path getFullPath(std::string const& _testFolder) const;
```

Comment:
These can stay `private` I guess.

Reply:
it is used to save the hive test file 
executeTest is gona be removed.

Reply:
OK.